package tests.chapter4;

import java.io.IOException;
import java.util.Scanner;

public class Test {
    public static void main(String[] args) throws Exception {

        try(Scanner s = new Scanner(System.in)){

        }
    }
}

